imgdoc2API/codecsAPI.h

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: 2024 Carl Zeiss Microscopy GmbH
+//
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "imgdoc2API.h"
+#include "importexport.h"
+#include "errorcodes.h"
+#include "bitmapinfointerop.h"
+#include "decodedimageresultinterop.h"
+
+/// Decodes the specific compressed data into an uncompressed bitmap. This destination bitmap is allocated
```

Comment:
(Opinion) Could be "Decodes the specified". Sounds more natural to me

Reply:
done, thanks